ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/model/jsonrpc/EeaSendRawTransaction.java

```diff
@@ -46,30 +48,39 @@ public String request() {
         eeaJsonRpc.eeaSendRawTransaction(
             "0xf90110a0e04d296d2460cfb8472af2c5fd05b5a214109c25688d3704aed5484f9a7792f28609184e72a0008276c094d46e8dd67c5d32be8058bb8eb970870f0724456780a9d46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f07244567536a00b528cefb87342b2097318cd493d13b4c9bd55bf35bf1b3cf2ef96ee14cee563a06423107befab5530c42a2d7d2590b96c04ee361c868c138b054d7886966121a6aa307837353737393139616535646634393431313830656163323131393635663237356364636533313464ebaa3078643436653864643637633564333262653830353862623865623937303837306630373234343536378a72657374726963746564");
     sendRawTransactionRequest.setId(77);
-
     return Json.encode(sendRawTransactionRequest);
   }
 
   @SuppressWarnings("unchecked")
   public String request(final Request<?, EthSendTransaction> request, final long chainId) {
-    final List<PrivateTransaction> params = (List<PrivateTransaction>) request.getParams();
-    final PrivateTransaction transaction = params.get(0);
+    final List<JsonObject> params = (List<JsonObject>) request.getParams();
+    if (params.size() != 1) {
+      throw new IllegalStateException("eeaSendTransaction request must have only 1 parameter");
+    }
+    final JsonObject transaction = params.get(0);
```

Comment:
Its just a nit pick, I would rename variable `transaction` to `transactionJson` ... which then make sense later as `transactionJson.getString("nonce")`. Its still OK though.